### PR TITLE
Early Audit Fixes

### LINKED ISSUE #16: Early Touchtable Audit
There are several UX/UI components that should be changed based on a 10/16/18 audit on the touch table. This includes

Classification Panel
- [x] Include button descriptions (ex. smooth + descriptions underneath)

Examples Panel
- [x] Examples panel should be open when first opening the classifier
- [x] Text content on some of the examples has changed
- [x] If an example category is selected (ex: smooth), it should reset if the classifier is closed

Leveler
- [x] Leveler should begin with one block filled in and five classifications leading to the next level
- [x] Level should close when classifier does

Classification Summary
- [x] Header takes up too much space

Close Workspace button
- [x] Workspace should be lowercase

### PR DESCRIPTION
**Pull request by wgranger**
Closes #16 

This PR closes issues associated with an early audit. Most of the changes here are minimal. However, there is a larger fix of introducing sub-labels to the classifier questions, which relies on italicized to be included in the question Markdown. (eg. Usually denoted by text within underscores).

### CODE REVIEW

**Comment by noahmalmed** [comment]
In `ClassificationPanelViewModel`, there seems to be a race condition with the `GraphQLClient`. 

The error is 'cannot perform runtime binding on a null reference' for the line `var data = reductions.First.data`

**Comment by wgranger** [comment]
Ahhh. This error you saw above is linked to the fact I changed the subjects on this workflow yesterday and the current ones don't have reductions from Caesar. Good to see this.

File: GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs [hunk 1]
```diff
@@ -30,6 +30,7 @@ public class ClassificationPanelViewModel : INotifyPropertyChanged
         public string CurrentTaskIndex { get; set; }
         public GraphQLHttpClient GraphQLClient { get; set; } = new GraphQLHttpClient("https://caesar-staging.zooniverse.org/graphql");
         public LevelerViewModel LevelerVM { get; set; } = new LevelerViewModel();
+        public ExamplesPanelViewModel ExamplesVM { get; set; } = new ExamplesPanelViewModel();
```
[the remark this answers is not in the document]
Comment (wgranger) [reply GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs]: Makes sense.

File: GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs [hunk 2]
```diff
@@ -306,16 +308,19 @@ private async void GraphQLRequest()
...
-            foreach (var count in data)
+            if (reductions.Count > 0)
             {
-                var index = System.Convert.ToInt32(count.Name);
-                AnswerButton test = CurrentAnswers[index];
+                var data = reductions.First.data;
+                foreach (var count in data)
+                {
+                    var index = System.Convert.ToInt32(count.Name);
+                    AnswerButton test = CurrentAnswers[index];
```
[the remark this answers is not in the document]
Comment (wgranger) [reply GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs]: No. It was initially named this as I was testing this approach and it needs to be changed.

**Comment by wgranger** [comment]
This should be good for another review. I also had to fix a bug where the counts on the classification summary were incorrectly persisting.

**Pull request merged**